CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Smithy Changelog
 
+## 1.46.0 (2024-03-19)
+
+### Features
+
+* Added protocol tests for `null` values in unions. ([#2180](https://github.com/smithy-lang/smithy/pull/2180))
+* Updated `ec2QueryName` tests to reflect usage. ([#2186](https://github.com/smithy-lang/smithy/pull/2186))
+* Updated model validation and protocol tests to use the new Gradle plugins. ([#2176](https://github.com/smithy-lang/smithy/pull/2176))
+* Added data-shape-only visitor class. ([#2168](https://github.com/smithy-lang/smithy/pull/2168))
+* Added protocol tests for S3 when dots are part of a key segment. ([#2166](https://github.com/smithy-lang/smithy/pull/2166))
+* Added the `@traitValidators` trait that can constrain shape closures. ([#2156](https://github.com/smithy-lang/smithy/pull/2156))
+* Added logic to infer default inline I/O suffixes in the IDL serializer. ([#2122](https://github.com/smithy-lang/smithy/pull/2122))
+* Added annotation processor base class for executing Smithy-Build plugins. ([#2073](https://github.com/smithy-lang/smithy/pull/2073))
+
+### Bug Fixes
+
+* Fixed a bug when creating `@standardRegionalEndpoints` nodes. ([#2179](https://github.com/smithy-lang/smithy/pull/2179))
+* Fixed build for JDK21 by adding CI matrices (and switches to Corretto), updates Gradle to 8.6 and Spotbugs to 6.0.8. ([#2171](https://github.com/smithy-lang/smithy/pull/2171))
```

Comment:
This can be removed, there's no public facing change for it.

Reply:
done